### PR TITLE
README fixes

### PR DESCRIPTION
**Pull request by sloria**
- Use `h.response()` instead of `reply()`
- Use `const` and arrow functions
- Fix a few invalid usages of callbacks

### CODE REVIEW

**Comment by sloria** [comment]
The build failure is unrelated to these changes. I was going to send a PR to fix the failures, but unfortunately, I cannot reproduce the test failures on node 9 (even using the same node and npm versions that Travis is using).

**Comment by Marsup** [comment]
Temporary travis hiccup, fixed by rebuilding. Changes LGTM, I'll let maintainer approve this one.

File: README.md [hunk 1]
```diff
@@ -48,9 +48,9 @@ const start = async () => {
         path: '/h',
         config: {
             id: 'hello',
-            handler: function (request, reply) {
+            handler: function (request, h) {
 
-                return reply('world!');
+                return h.response('world!');
```
Comment (hueniverse) [inline README.md]: `return 'world!';`

File: README.md [hunk 2]
```diff
@@ -77,35 +77,33 @@ client.connect().then(() => {
...
-var Hapi = require('hapi');
-var Nes = require('nes');
+const Hapi = require('hapi');
+const Nes = require('nes');
 
-var server = new Hapi.Server();
+const server = new Hapi.Server();
 
-server.register(Nes).then(() => {
+server.register(Nes).then(async () => {
```
Comment (hueniverse) [inline README.md]: I rather switch this to use await. Basically get rid of all the `then()` in the examples.

Comment (sloria) [reply README.md]: OK, should I do this for the client examples as well?

**Comment by sloria** [comment]
@hueniverse OK, updated.

File: README.md [hunk 3]
```diff
@@ -62,100 +61,105 @@ start();
...
 ```js
-var Nes = require('nes');
-
-var client = new Nes.Client('ws://localhost');
-client.connect().then(() => {
-    var handler = function (update, flags) {
+const Nes = require('nes');
 
+const client = new Nes.Client('ws://localhost');
+const start = async () => {
```
Comment (sloria) [inline README.md]: @hueniverse If you prefer, I could also make these IIFEs, e.g.

```js
(async () => {
    await client.connect();
})()
```
Let me know your preference.

**Pull request merged**